packages/instant/src/util/signed_order_coercion.ts

```diff
@@ -0,0 +1,26 @@
+import { SignedOrder } from '@0x/types';
```

Comment:
I would structure this similar to https://github.com/0xProject/0x-monorepo/blob/development/packages/order-utils/src/parsing_utils.ts#L4

So have a general function that tries to convert chosen keys to BigNumber and then define the fields in a separate function. This logic does belong here though as it is pretty specific to instant.